frontend/packages/core/src/NPS/banner.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+
+import type { EmojiType } from "../Assets/emojis";
+import { HappyEmoji } from "../Assets/emojis";
+import { Button } from "../button";
+import { useAppContext } from "../Contexts";
+import { FeatureOn, SimpleFeatureFlag } from "../flags";
+import Grid from "../grid";
+import Paper from "../paper";
+import styled from "../styled";
+import { Typography } from "../typography";
+
+export interface BannerFeedbackProps {
+  /**
+   * Whether this component should appear integrated into the page versus in its own container
+   * @defaultValue false
+   */
+  integrated?: boolean;
```

Comment:
👍 should we call this something like `elevated` to mirror the prop naming for material's paper?

Reply:
@jdslaugh what do you think about the naming here?

Reply:
Totally missed this comment, whoops. Didn't actually realize MUI had a prop for elevated, yeah I think that makes sense since elevated=0 is basically what we're achieving here. Let me adjust

Reply:
Adjusted to have elevated be the default now, and false would be what used to be integrated.